src/game/AI/ScriptDevAI/scripts/eastern_kingdoms/redridge_mountains.cpp

```diff
@@ -105,23 +85,7 @@ struct npc_corporal_keeshan_escortAI : public npc_escortAI
     {
```

Comment:
Delete whole UpdateAI func else spell lists wont work. The base escort ai code will work perfectly fine with no changes.

Reply:
done